qemu/tests/cfg/vm_hugetlb_selftests.cfg

```diff
@@ -0,0 +1,7 @@
+- vm_hugetlb_selftests:
+    only Linux
+    type = vm_hugetlb_selftests
+    virt_test_type = qemu
+    setup_hugepages = yes
+    depends_pkgs = rsync libcap-devel numactl-devel glibc-devel.*i686
```

Comment:
I guess this is only for x86_64, as I see the i686 suffix, if that is true, we can add only x86_64 filter